pkg/operator/ceph/cluster/osd/osd_test.go

```diff
@@ -570,22 +570,18 @@ func TestDetectCrushLocation(t *testing.T) {
 
 	// update the location with valid topology labels
 	nodeLabels = map[string]string{
-		"failure-domain.beta.kubernetes.io/region": "region1",
-		"failure-domain.beta.kubernetes.io/zone":   "zone1",
```

Comment:
Instead of removing the region and zone from this test, how about adding them back in the supported label? for example: `topology.kubernetes.io/zone`